### DISCUSSION TITLE
AssemblyDependencyResolver

### DISCUSSION THREAD

**Discussion by TartanLeGrand** [dotnet/maui#5547] [Q&A]
```csharp
[System.Runtime.Versioning.UnsupportedOSPlatform("android")]
[System.Runtime.Versioning.UnsupportedOSPlatform("browser")]
[System.Runtime.Versioning.UnsupportedOSPlatform("ios")]
[System.Runtime.Versioning.UnsupportedOSPlatform("tvos")]
public sealed class AssemblyDependencyResolver
```

AssemblyDependencyResolver is marked as not supported by : 
- android
- browser
- ios
- tvos

Is there a replacement or another way to use it?

**Comment by ThaDaVos** [comment]
Here for the same question - even doing a NativeAoT compilation of a DotNet 9 project which is specifically targeted to `win-x64` with the `-r` parameter will throw the `Operation is not supported on this platform` message - really curious if there's any way to create an application with Plugin support for other Platforms then Windows or for NativeAoT...

Followed guide at: https://learn.microsoft.com/en-us/dotnet/core/tutorials/creating-app-with-plugin-support

**Reply by ThaDaVos** [reply]
Created a feature request: https://github.com/dotnet/sdk/issues/42389
